fedgov-cv-rgaencoder/src/main/java/gov/usdot/cv/rgaencoder/DDate.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2024 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package gov.usdot.cv.rgaencoder;
+
+public class DDate {
+    private Integer year;
```

Comment:
Be consistent in using int or Integer throughout this file.

Note:  Integer is an object.. Creating instances of it involves memory allocation and, eventually, garbage collection when the objects are no longer needed. In contrast, int is a primitive data type, and its memory allocation and access are faster because it doesn't involve object overhead.